pkg/web_css/lib/src/_variables.scss

```diff
@@ -16,17 +16,19 @@
 //   - `[property]` may be `color`, `opacity` for specific values, or a `value` for multi-part properties.
 
 :root {
+  @include dash_variables.light-theme;
+
   --pub-color-white:         #ffffff;
   --pub-color-snowWhite:     #fafafa; // slight deviation from official snow-white (fffafa)
   --pub-color-smokeWhite:    #f5f5f7; // slight deviation from official white-smoke (f5f5f5)
   --pub-color-bubblesBlue:   #e7f8ff; // slight deviation from official bubbles-blue (e7feff)
-  --pub-color-gunpowderGray: #4a4a4a; // hsl(0, 0%, 29%);
+  // --pub-color-gunpowderGray: #4a4a4a; // hsl(0, 0%, 29%);
```

Comment:
Should this just be deleted?

Reply:
I'd rather keep this for a short while, until we have a better consolidated view on how the naming + shared naming will work. But at the same time I can also just delete it.

Reply:
As you prefer - just note that the usual policy is to not leave commented out code.

You can always find it in git history.